warehouse/templates/manage/releases.html

```diff
@@ -40,12 +40,18 @@ <h2>Releases ({{ project.releases|length }})</h2>
           </th>
           <td class="table__date">{{ humanize(release.created) }}</td>
           <td class="table__summary">
-            {% if release.summary %}
-              {{ release.summary }}
+            {% set num_files = release.files.all()|length %}
```

Comment:
@di is this likely to create performance issues?  i.e. will it hit the db?

Reply:
Yes. I added a commit here which will do this more efficiently, perhaps @woodruffw can add the necessary tests?

As a side effect, we're able to say how many of each packagetype for a given release, so I added that, e.g. instead of:

> 7 files (Wheel, Source)

it will say:

> 7 files (6 Wheel, 1 Source)

feel free to change as necessary, but I think this is more clear.

Reply:
Great, thanks Dustin :)